### PR TITLE
Switch to ocean polygons

### LINKED ISSUE #1982: Switch to ocean polygons
We should switch from preprocessed land polygons on a water colour background to preprocessed water polygons on a land colour background.

This would
- Allow easier customization with hillshading
- Be quicker in inland areas
- Allow more water styling, including outlines of water
- Allow rendering on top of maritime borders, making it possible to have them less prominent
- Makes it easier to render inland areas with no coastline by substituting an empty shapefile

Cross-ref #621 

Issues
- There are no simplified water polygons

cc @joto @imagico

### PR DESCRIPTION
**Pull request by pnorman**
This should have no rendering changes, as the base land and base ocean are still the first things drawn.

We may want to re-order later to put water polygons by ocean polygons, but this will require consideration of what landuse should appear on oceans

Fixes #1982.

### CODE REVIEW

**Comment by pnorman** [comment]
cc @imagico

**Comment by imagico** [comment]
This addresses #1982.  For #1547 changing layer order to render ocean polygons directly above/below water areas will be good but that can be considered a separate problem.

File: INSTALL.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ To download the shapefiles you can run the following script from this directory.
 You can also download them manually at the following paths:
 
 * [`simplified-land-polygons.shp`](http://data.openstreetmapdata.com/simplified-land-polygons-complete-3857.zip) (updated daily)
```
Comment (sommerluk) [inline INSTALL.md]: Can this line be removed?

File: INSTALL.md [hunk 2]
```diff
@@ -25,7 +25,9 @@ To download the shapefiles you can run the following script from this directory.
...
+* [`simplified-water-polygons.shp`](http://data.openstreetmapdata.com/simplified-water-polygons-complete-3857.zip) (updated daily)
 * [`land-polygon.shp`](http://data.openstreetmapdata.com/land-polygons-split-3857.zip) (updated daily)
```
Comment (sommerluk) [inline INSTALL.md]: Can this line be removed?

**Comment by math1985** [comment]
Thanks @pnorman!

**Comment by math1985** [comment]
Unfortunately, I had to revert this change for now as it's still not fully clear what causes #2101. See also #2174.

I have created a branch ocean-polygons which can be used for testing the problem (this branch is not intended to be released, it is only there to make testing easier).

As soon as #2101 is sorted out, we can again try to move to ocean polygons.

**Comment by math1985** [comment]
I have reverted this PR for now, see #2101.

**Pull request merged**